### PR TITLE
Copyright script

### PR DESCRIPTION
**Pull request by trenatov**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre-alpine
+FROM openjdk:12.0.2-jdk-oracle
```
Comment (ruslanys) [inline docker/Dockerfile:1]: we use lts jdks only

File: build.gradle [hunk 2]
```diff
@@ -12,7 +12,8 @@ plugins {
 apply plugin: 'io.spring.dependency-management'
 
 group = 'io.openfuture.api'
-version = versionDetails().lastTag[1..-1] + (versionDetails().isCleanTag ? '' : '-DIRTY')
+// Temporary solution
+version = 'v2.6.9'   //versionDetails().lastTag[1..-1] + (versionDetails().isCleanTag ? '' : '-DIRTY')
```
Comment (ruslanys) [inline build.gradle:16]: use `id 'io.zensoft.versioning' version '1.1.0'` plugin instead

File: docker/Dockerfile (hunk 1 above)
Comment (ruslanys) [inline docker/Dockerfile:1]: we use lts jdks only, switch to 11.

Also we do not use proprietary Oracle builds.

**Review by ruslanys — CHANGES_REQUESTED** [review]

**Comment by trenatov** [comment]
Will be created from master branch

**Pull request closed without merging**